packages/commonwealth/client/scripts/views/components/Polls/PollCard/PollCard.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import {
+  CastVoteProps,
+  CastVoteSection,
+  DeletePollModal,
+  PollOptionProps,
+  PollOptions,
+  ResultsSectionProps,
+  ResultsSections,
+  VoteDisplay,
+} from 'views/components/Polls';
+import { buildVoteDirectionString } from 'views/components/Polls/utils';
+import { CWCard } from 'views/components/component_kit/cw_card';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWModal } from 'views/components/component_kit/new_designs/CWModal';
+
+import './PollCard.scss';
+
+export type PollCardProps = PollOptionProps &
+  CastVoteProps &
+  ResultsSectionProps & {
+    hasVoted?: boolean;
+    incrementalVoteCast?: number;
+    proposalTitle?: string;
+    showDeleteButton?: boolean;
+    onDeleteClick?: () => void;
+  };
+
+export const PollCard = ({
+  disableVoteButton = false,
+  isPreview = false,
+  showDeleteButton = false,
+  onDeleteClick,
+  multiSelect,
+  onResultsClick,
+  onVoteCast,
+  pollEnded,
+  proposalTitle,
+  timeRemaining,
+  tokenSymbol,
+  tooltipErrorMessage,
+  votedFor,
+  voteInformation,
+  hasVoted,
+  totalVoteCount,
+}: PollCardProps) => {
+  const [selectedOptions, setSelectedOptions] = useState<Array<string>>([]);
+  const [deleteModalOpen, setDeleteModalOpen] = useState(false);
+
+  const resultString = 'Results';
+
+  const castVote = () => {
+    if (
+      multiSelect ||
+      selectedOptions[0] === votedFor ||
+      selectedOptions.length === 0
+    ) {
+      // TODO: Build this out when multiple vote options are introduced.
```

Comment:
is this for a followup pr or for some other time

Reply:
as stated in the PR description I just copied over a lot of code to the new files. I am not able to go through all those files and clean them up/refactor as this is whole new scope of work.

> because of the CI errors, I moved polls components to separate directories in [this commit](https://github.com/hicommonwealth/commonwealth/pull/8883/commits/86d2b210abfe720bddb7733ea7eb616059aeb5a9). It might contain some legacy code but the intention of this PR is not doing hard refactor of the polls code

Reply:
okay so multi select  was probably something that team was planning to develop but it never happened. I just removed code related to multi-select